benchmarks/tracer/scenario.py

```diff
@@ -0,0 +1,36 @@
+import pyperf
+
+from ddtrace import tracer
+# should I have the traced variant be True or False? Or should I just directly pass in the function via the variant?
+VARIANTS = [
+    dict(traced=True, calls=100),
+    dict(traced=False, calls=100),
```

Comment:
I'm not sure if we need a untraced variant here since we are interesting in timing the tracer.